Intersect.Client.Core/Interface/Game/Inventory/InventoryItem.cs

```diff
@@ -269,72 +258,55 @@ private void _iconImage_DoubleClicked(Base sender, MouseButtonState arguments)
         }
     }
 
-    private void _iconImage_Clicked(Base sender, MouseButtonState arguments)
+    private void IconImage_Clicked(Base sender, MouseButtonState arguments)
     {
-        switch (arguments.MouseButton)
+        if (arguments.MouseButton is MouseButton.Right)
```

Comment:
Since this is the only one in this method, flip it and return if it's not Right to reduce nesting